src/Ecal/EcalVetoProcessor.cxx

```diff
@@ -520,6 +808,9 @@ void EcalVetoProcessor::produce(framework::Event &event) {
     result.setDiscValue(pred);
     // std::cout << "  pred > bdtCutVal = " << (pred > bdtCutVal_) << std::endl;
 
+    bool passesTrackingVeto =
+        (nStraightTracks_ == 0) && (nLinregTracks_ == 0) &&
+        (firstNearPhLayer_ >= 6) && (epAng_ > 3.0 || epSep_ > 10.0);
```

Comment:
This variable `passesTrackingVeto` is not used anywhere. Should it be added as another member of the EcalVetoResult? Should it be factored into whether the event passes the ecal veto?

Reply:
It depends on whether you think it's worth keeping track independently of the overall veto result.  Currently, it's just used for the veto result, so the track cuts could be stuffed into the result.setVetoResult() expression instead.

Reply:
I think it should be included. If we end up separating the mip tracking from the BDT, we should make separate event objects for them so downstream users understand that they are separate. For clarity, I would do something like
```cpp
   bool passesTrackingVeto =
        (nStraightTracks_ == 0) && (nLinregTracks_ == 0) &&
        (firstNearPhLayer_ >= 6) && (epAng_ > 3.0 || epSep_ > 10.0);
   bool passesBDTVeto = pred > bdtCutVal_
   result.setVetoResult(passesBDTVeto && passesTrackingVeto);
```
This just shows at a quick glance that you are combining the BDT and tracking vetos to make the final veto decision.

Reply:
Okay--I just added that in the latest changes.